src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostUnclaimByIdControllerTest.java

```diff
@@ -33,55 +40,176 @@ public void should_return_a_404_if_task_does_not_exist() {
                 .format(DateTimeFormatter.ofPattern(DATE_TIME_FORMAT))))
             .body("error", equalTo(HttpStatus.NOT_FOUND.getReasonPhrase()))
             .body("status", equalTo(HttpStatus.NOT_FOUND.value()))
-            .body("message",
-                equalTo("There was a problem unclaiming the task with id: " + nonExistentTaskId));
+            .body("message", equalTo(
+                String.format("There was a problem fetching the task with id: %s", nonExistentTaskId)));
     }
 
     @Test
-    public void should_return_a_404_when_unclaiming_a_task_by_id_with_different_credentials() {
-
+    public void should_return_a_403_when_the_user_did_not_have_any_roles() {
 
         Map<String, String> task = common.setupTaskAndRetrieveIds();
 
+        Response result = restApiActions.post(
+            ENDPOINT_BEING_TESTED,
+            task.get("taskId"),
+            authorizationHeadersProvider.getLawFirmAAuthorization()
+        );
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.FORBIDDEN.value())
+            .contentType(APPLICATION_JSON_VALUE)
+            .body("timestamp", lessThanOrEqualTo(LocalDateTime.now()
+                                                     .format(DateTimeFormatter.ofPattern(DATE_TIME_FORMAT))))
+
+            .body("error", equalTo(HttpStatus.FORBIDDEN.getReasonPhrase()))
+            .body("status", equalTo(HttpStatus.FORBIDDEN.value()))
+            .body("message", equalTo("User did not have sufficient permissions to perform this action"));
+    }
+
+    @Test
+    public void should_return_a_204_when_unclaiming_a_task_by_id() {
+        Headers headers = authorizationHeadersProvider.getTribunalCaseworkerAAuthorization();
+
+        Map<String, String> task = setupScenario(headers);
+
+        Response result = restApiActions.post(
+            ENDPOINT_BEING_TESTED,
+            task.get("taskId"),
+            headers
+
+        );
+        result.then().assertThat()
+            .statusCode(HttpStatus.NO_CONTENT.value());
+
+        assertions.taskVariableWasUpdated(task.get("taskId"), "taskState", "unassigned");
+
+
+    }
+
```

Comment:
We need FTs to cover the other 204 scenarios for different permissions. See claim PostClaimByIdControllerTest 204(tests)